tests/drum/test_fit.py

```diff
@@ -199,6 +203,59 @@ def test_fit(
             cmd, "Failed in {} command line! {}".format(ArgumentsOptions.MAIN_COMMAND, cmd)
         )
 
+    @pytest.mark.parametrize(
+        "framework, problem, docker, parameters",
+        [
+            (SKLEARN_BINARY_HYPERPARAMETERS, BINARY_TEXT, None, SKLEARN_BINARY_PARAMETERS),
+            (SKLEARN_BINARY_HYPERPARAMETERS, BINARY_SPACES, None, SKLEARN_BINARY_PARAMETERS),
+            (SKLEARN_TRANSFORM_HYPERPARAMETERS, REGRESSION, None, SKLEARN_TRANSFORM_PARAMETERS),
+        ],
+    )
+    @pytest.mark.parametrize("weights", [WEIGHTS_CSV, WEIGHTS_ARGS, None])
+    def test_fit_hyperparameters(
+        self, resources, framework, problem, docker, parameters, weights, tmp_path,
+    ):
+        if framework == RDS:
```

Comment:
Are you going to make an R template? 

Also, can you clean up this function if you're not going to use the various logic branches?

Reply:
Eventually, and sure